packages/typespec-ts/test/integration/generated/client/naming/src/clientDefinitions.ts

```diff
@@ -39,19 +39,19 @@ export interface Parameter {
 
 export interface PropertyClient {
   post(
-    options?: PropertyClientParameters,
+    options: PropertyClientParameters,
```

Comment:
https://github.com/Azure/cadl-ranch/commit/fd61404ddd3d374be328af588b11985af4869189#diff-84598cc2d97c5c2a7f950a9c4d97316840c05c6f611030c2f97d812e9480b345 it should be expected.